custom-metrics-autoscaling/direct-to-sd/README.me

```diff
@@ -0,0 +1 @@
+A simple sd-dummy-exporter container exports a metric of constant value to Stackdriver in a loop. The metric name and value can be passed by flags. Pod ID is also passed by a flag.
```

Comment:
should be .md, also add a heading using:

```md
# Example Name

Description description  description  description...
```

Reply:
Done